File: AMS_基础信息组件/code/server/source/com.yss.ams.base.information.support/src/main/java/com/yss/ams/base/information/support/sys/portbusinessrange/pojo/PortBusinessRangePojo.java
/**
 *
 * @Title: BusinessType.java 
 * @Package com.yss.ams.base.information.support.sys.businesstype.pojo 
 * @date 2019年5月13日 下午3:30:19 
 * @version V1.0
 * @Stroy/Bug
 * @author xiadeqi   
 */
package com.yss.ams.base.information.support.sys.portbusinessrange.pojo;

import com.yss.framework.api.common.co.AuditableParamPojo;

/** 
 * 组合业务关联关系pojo类
 * @ClassName: BusinessType 
 * @date 2019年5月13日 下午3:30:19
 * @Stroy72335/Bug
 * @author xiadeqi 
 */
public class PortBusinessRangePojo extends AuditableParamPojo{

	/** 
	 * @Fields serialVersionUID : TODO(用一句话描述这个变量表示什么) 
	 */ 
	private static final long serialVersionUID = -3520008156412630800L;
	/**
	 * 组合代码
	 */
	private String c_PORT_CODE = "";
	/**
	 * 业务类型代码
	 */
	private String c_BUSINESS_CODE = "";
	public String getC_PORT_CODE() {
		return c_PORT_CODE;
	}
	public void setC_PORT_CODE(String c_PORT_CODE) {
		this.c_PORT_CODE = c_PORT_CODE;
	}
	public String getC_BUSINESS_CODE() {
		return c_BUSINESS_CODE;
	}
	public void setC_BUSINESS_CODE(String c_BUSINESS_CODE) {
		this.c_BUSINESS_CODE = c_BUSINESS_CODE;
	}
	
}

File: AMS_基础信息组件/code/client/source/YssInformation.Support/Sys/PortBusinessRange/Pojo/PortBusinessRangePojo.cs
using FAST.Common.Service.Pojo.Base;
using Newtonsoft.Json;

namespace YssInformation.Support.Sys.PortBusinessRange.Pojo
{
    /// <summary>
    /// 业务类型关联组合pojo类
    /// </summary>
    public class PortBusinessRangePojo : AuditableParamPojo
    {
        /// <summary>
        /// 组合代码
        /// </summary>    
        private string c_PORT_CODE = "";

        /// <summary>
        /// 业务类型
        /// </summary>
        private string c_BUSINESS_CODE = "";

        /// <summary>
        /// 属性：组合代码
        /// </summary>
        [JsonProperty(PropertyName = "c_PORT_CODE")]
        public string C_PORT_CODE
        {
            set { c_PORT_CODE = value; }

            get { return c_PORT_CODE; }
        }

        /// <summary>
        /// 属性：业务类型
        /// </summary>
        [JsonProperty(PropertyName = "c_BUSINESS_CODE")]
        public string C_BUSINESS_CODE
        {
            set { c_BUSINESS_CODE = value; }

            get { return c_BUSINESS_CODE; }
        }
    }
}

